tests/handlers/koji/test_rebuild_flatpak_application_on_module_ready.py

```diff
@@ -178,5 +180,172 @@ def test_event_state_updated_when_no_images_with_higher_rpm_nvr(self):
         self.assertEqual(db_event.state, EventState.SKIPPED.value)
         self.assertEqual(
             db_event.state_reason,
-            "There is no image can be rebuilt. message_id: 123",
+            "No images are impacted by the advisory.  message_id: 123",
         )
+
+    def test_get_module_name_stream_set(self):
+        build = {
+            "build_id": 1850907,
+            "epoch": None,
+            "extra": {
+                "typeinfo": {
+                    "module": {
+                        "modulemd_str": '---\ndocument: modulemd\nversion: 2\ndata:\n  name: ghc\n  stream: "9.2"\n  version: 3620211101111632\n  context: d099bf28\n  summary: Haskell GHC 9.2\n  description: >-\n    This module provides the Glasgow Haskell Compiler version 9.2.1\n',
```

Comment:
could you put this string on multiple lines for readability?

Reply:
Ok. Fixed.